edgelet/iotedge/src/check/checks/up_to_date_config.rs

```diff
@@ -0,0 +1,39 @@
+use failure::{self, Context};
+
+use aziotctl_common::check_last_modified::check_last_modified;
+use aziotctl_common::check_last_modified::CheckResult as InternalCheckResult;
+
+use crate::check::{checker::Checker, Check, CheckResult};
+
+#[derive(Default, serde_derive::Serialize)]
+pub(crate) struct UpToDateConfig {}
+
+impl Checker for UpToDateConfig {
+    fn id(&self) -> &'static str {
+        "iotedge-config-up-to-date"
+    }
+    fn description(&self) -> &'static str {
+        "iotedge configuration up-to-date with config.toml"
```

Comment:
For consistency with the others, no "iotedge"

Reply:
Fixed